src/Components/ImageCommands/index.js

```diff
@@ -0,0 +1,31 @@
+import React from 'react';
+import { ClipboardCopy, Stack, StackItem } from '@patternfly/react-core';
+import { imageProps } from '../ProvisioningWizard/helpers';
+
+const launchInstanceCommand = (options) => {
+  return `gcloud compute instances create ${options.image_name}-instance --image-project ${options.project_id} --image ${options.image_name}`;
+};
+
+const ImageCommand = ({ image }) => {
+  return (
+    <Stack>
```

Comment:
This is already mounted in a `Stack` layout  in `SourceMissing` component, instead of having two stacks, instead you can wrap these `StackItems` with `<>` (react fragment)